gpiozero/output_devices.py

```diff
@@ -236,7 +240,8 @@ def off(self):
         self._stop_blink()
         self._write(False)
 
-    def blink(self, on_time=1, off_time=1, n=None, background=True):
+    def blink(self, on_time=1, off_time=1, end_state=None,
```

Comment:
I think for backwards-compatibility reasons, we prefer to not reorder the args-list. So here you'd need to move `end_state` to after `background` .

Reply:
I didn't think about that. No problem!